rtgui/imagearea.cc

```diff
@@ -618,19 +611,18 @@ void ImageArea::initialImageArrived (CropWindow* cw)
 
     if (mainCropWindow) {
         if(firstOpen || options.prevdemo != PD_Sidecar || (!options.rememberZoomAndPan) ) {
-            mainCropWindow->zoomFit (false);
+            mainCropWindow->zoomFit ();
             firstOpen = false;
             mainCropWindow->cropHandler.getFullImageSize(fullImageWidth, fullImageHeight);
         } else {
             int w, h;
             mainCropWindow->cropHandler.getFullImageSize(w, h);
 
-            if(w == fullImageWidth && h == fullImageHeight) { // && mainCropWindow->getZoom() != mainCropWindow->getZoomFitVal()) {
-                int x, y;
-                mainCropWindow->getCropPosition(x, y);
-                mainCropWindow->setCropPosition(x, y, false);
+            if(w != fullImageWidth || h != fullImageHeight) {
+                printf("zoomFit !\n");
```

Comment:
Yes, it's a left over, deleted.

As you said, printf has its uses. I'm not very comfortable with debugging on Windows using Linux tools. It's slow, and sometimes hang.